pcdsdevices/pim.py

```diff
@@ -334,10 +366,17 @@ class XPIMFilterWheel(StatePositioner):
 
     tab_component_names = True
 
-    state = Cpt(EpicsSignal, ':GET_RBV', write_pv=':SET', kind='normal')
+    state = Cpt(EpicsSignal, ':GET_RBV', write_pv=':SET', kind='normal',
+                doc='Control of the filter wheel state by preset '
+                    'transmission percentages.')
+
+    reset_cmd = Cpt(PytmcSignal, ':ERR:RESET', io='i', kind='omitted',
+                    doc='Command to reset a filter wheel error.')
+    error_message = Cpt(PytmcSignal, ':ERR:MSG', io='i', kind='omitted',
+                        string=True,
```

Comment:
Good catch :+1:

Reply:
It's one of a handful that looks very silly in typhos right now because it is an array instead of a string. Should be better after this.